openiit.html

```diff
@@ -109,8 +109,8 @@ <h3 style="text-align: center"><strong><u>Open IIT Results</u></strong></h3>
                 .then(() => {
                     document.getElementById('openiitBody').innerHTML = output;
                 })
-                .catch(() => {
-                    document.getElementById('openiitBody').innerHTML = `<tr><td colspan=7><h1 style="text-align:center;">To Be Updated</h1></td></td>`;
+                .catch(err => {
+                    console.log('Error is ', err);
```

Comment:
Along with logging the error, we should keep the "To Be Updated" tag since we do not want people to know that there was an error.

Reply:
In case of error don't you think it would be wrong to show that table is about to get updated instead of telling that there  was an errror server side